### PR TITLE
GRVDEV-91 Create AWS Static Website module

### PR DESCRIPTION
**Pull request by TysonBeck**
Create secure s3 bucket for static website
Create CDN distribution

### CODE REVIEW

File: aws/s3-cdn/module.tf [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+variable "name" {
+  type        = string
+  default     = "s3-cdn"
+  description = "The name of the module"
+}
+
+variable terraform_module {
+  type        = string
+  default     = "gravicore/terraform-gravicore-modules/aws/cdn"
```
Comment (jeremyodle) [inline aws/s3-cdn/module.tf]: `aws/s3-cdn`

File: aws/s3-cdn/s3-cdn.tf [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+    }
+  }
+
+  web_acl_id          = var.web_acl_id
+  wait_for_deployment = var.wait_for_deployment
+
+  tags = local.tags
+}
+
+module "dns" {
```
Comment (jeremyodle) [inline aws/s3-cdn/s3-cdn.tf:505]: Are all resources/modules that support tags/comments have them associated?

Comment (TysonBeck) [reply aws/s3-cdn/s3-cdn.tf:505]: Yes, double checked and all supported resources have tags/comments/descriptions

File: aws/s3-cdn/module.tf (hunk 1 above)
Comment (TysonBeck) [reply aws/s3-cdn/module.tf]: Corrected

**Review by jeremyodle — APPROVED** [review]

**Pull request merged**